sdk/eventhub/Azure.Messaging.EventHubs/src/Producer/EventHubProducerClient.cs

```diff
@@ -35,6 +34,9 @@ namespace Azure.Messaging.EventHubs.Producer
     ///
     public class EventHubProducerClient : IAsyncDisposable
     {
+        /// <summary>The maximum number of attempts to send an event by a partition specific producer</summary>
```

Comment:
nit: missing period `.` at the end.

Reply:
We removed this constant in favour of picking it from the ```RetryPolicy```